full-moon/src/tokenizer.rs

```diff
@@ -108,6 +108,10 @@ pub enum Symbol {
     #[cfg_attr(feature = "serde", serde(rename = ">"))]
     GreaterThan,
 
+    #[cfg(feature = "lua53")]
+    #[cfg_attr(feature = "serde", serde(rename = ">>"))]
+    RightShift,
```

Comment:
I don't know if we should label `>>` as RightShift (and `<<` as LeftShift), as it seems most symbols aren't named after the function they provide (which makes sense, since if e.g. Luau introduces `>>` which isn't a RightShift operator)

The only other name I could think of is `DoubleGreaterThan`

Reply:
DoubleGreaterThan is better IMO